SingularityUI/app/views/expandableTableSubview.coffee

```diff
@@ -25,6 +25,7 @@ class ExpandableTableSubview extends View
 
     initialize: ({@collection, @template}) ->
         @listenTo @collection, 'sync', @render
+        @listenTo @collection, 'reset', @render
```

Comment:
It doesn't look like you can do multiple events in the same listenTo call. simpleSubview does it in a loop: 
```
  for eventName in ['sync', 'add', 'remove', 'change', 'reset']
    @listenTo @data, eventName, @render
```
I'll switch it to that. Though a bit overkill for now, it may be less confusing for anyone adding another.